gemfiles/rails_7_0.gemfile

```diff
@@ -0,0 +1,14 @@
+# This file was generated by Appraisal
+
+source "http://rubygems.org"
```

Comment:
Any chance to run `appraisal`? It should switch to `https` in all the gemfiles

Reply:
Shouldn't `appraisal` run as part of the CI?

Reply:
It should not be a problem on the CI, but if you run `appraisal` on top of this branch with a clean git status, you get the following diff:

```diff
diff --git a/gemfiles/rails_6_0.gemfile b/gemfiles/rails_6_0.gemfile
index 366dd67..81f1115 100644
--- a/gemfiles/rails_6_0.gemfile
+++ b/gemfiles/rails_6_0.gemfile
@@ -1,10 +1,10 @@
 # This file was generated by Appraisal
 
-source "http://rubygems.org"
+source "https://rubygems.org"
 
 gem "rails", "~> 6.0.0"
 gem "rspec"
-gem "rspec-rails", "~> 3.7"
+gem "rspec-rails", "~> 5.0"
 gem "sqlite3"
 gem "timecop"
 gem "byebug"
diff --git a/gemfiles/rails_6_1.gemfile b/gemfiles/rails_6_1.gemfile
index 3f2adc7..7d1f741 100644
--- a/gemfiles/rails_6_1.gemfile
+++ b/gemfiles/rails_6_1.gemfile
@@ -1,10 +1,10 @@
 # This file was generated by Appraisal
 
-source "http://rubygems.org"
+source "https://rubygems.org"
 
 gem "rails", "~> 6.1.0"
 gem "rspec"
-gem "rspec-rails", "~> 3.7"
+gem "rspec-rails", "~> 5.0"
 gem "sqlite3"
 gem "timecop"
 gem "byebug"
diff --git a/gemfiles/rails_edge.gemfile b/gemfiles/rails_edge.gemfile
index 1cb6e4f..0657570 100644
--- a/gemfiles/rails_edge.gemfile
+++ b/gemfiles/rails_edge.gemfile
@@ -1,10 +1,10 @@
 # This file was generated by Appraisal
 
-source "http://rubygems.org"
+source "https://rubygems.org"
 
 gem "rails", git: "https://github.com/rails/rails.git", branch: "main"
 gem "rspec"
-gem "rspec-rails", "~> 3.7"
+gem "rspec-rails", "~> 5.0"
 gem "sqlite3"
 gem "timecop"
 gem "byebug"
```

Also, `gemfiles/rails_7_0.gemfile` is not being generated by Appraisal because it is not listed